### ISSUE TITLE
QR Code can not be scanned on iphone if the code is 1.5 to 1.5 cm. 

### ISSUE THREAD

**Issue by arnold-b** [mebjas/html5-qrcode#826]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by ManuelLopezAnido** [comment]
Yes in this problem are talking about it. https://github.com/mebjas/html5-qrcode/issues/820
